### PR TITLE
ci: freeze 1.81 for near-workspaces paths (temporarily)

### PR DESCRIPTION
**Pull request by dj8yfo**
https://github.com/near/near-sdk-rs/blob/master/.github/workflows/test.yml#L19-L23 is not affected, because `Test Core` job uses [`near_workspaces::compile_project`](https://github.com/near/near-sdk-rs/blob/master/near-sdk/tests/store_performance_tests.rs#L58)

### CODE REVIEW

File: near-sdk/tests/store_performance_tests.rs [hunk 1]
```diff
@@ -221,6 +221,7 @@ async fn iter() -> anyhow::Result<()> {
     Ok(())
 }
 
+#[allow(clippy::ifs_same_cond)]
```
Comment (dj8yfo) [inline near-sdk/tests/store_performance_tests.rs:224]: this is a false lint that `since(1.82)` and `since(1.81)` are the same condition

**Comment by dj8yfo** [comment]
@race-of-sloths include

**Comment by race-of-sloths** [comment]
@dj8yfo Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
Weekly streak is on the road, smart strategy! Secure your streak with another PR!

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://race-of-sloths.com/images/bot-banner-dark.svg">
    <source media="(prefers-color-scheme: light)" srcset="https://race-of-sloths.com/images/bot-banner-light.svg">
    Shows inviting banner with latest news.
</picture>](https://race-of-sloths.com/latest-news)

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/dj8yfo?type=bot&pr=near/near-sdk-rs/1250&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/dj8yfo?type=bot&pr=near/near-sdk-rs/1250&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/dj8yfo)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command. Alternatively, autoscoring [1,2] will be applied for this pull request

</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/dj8yfo)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Invite contributor** `@race-of-sloths invite` to invite the contributor to participate in a race or include it, if it's already a runner.
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Comment by race-of-sloths** [comment]
🥁  __Score it!__

@frol, please score the PR with `@race-of-sloths score [1/2/3/5/8/13]`. The contributor deserves it.
If no scoring is provided within 24 hours, this PR will be scored as 2 🦥

**Comment by frol** [comment]
@race-of-sloths score 3

**Comment by race-of-sloths** [comment]
### ⏰️ Scoring closed
24 hours have already passed since the merge

**Pull request merged**